### PR TITLE
Add support for additional URL parameters

### PR DESCRIPTION
**Pull request by chrs-myrs**

### CODE REVIEW

**Review by xyNNN — APPROVED** [review]
Looks very good. Nothing to mention!

**Comment by xyNNN** [comment]
One thing: I've seen that tracis-ci.org was ceased and released under travis-ci.com. All projects which was hosted under *.org are now in read-only mode. I've migrated to travis-ci.com and enabled all hooks. Could you please do a test commit, to see if the CI is running great. After this I can merge this and tag a new release!

Thanks @chrs-myrs  for your work!

**Comment by chrs-myrs** [comment]
Sorry for the delay @xyNNN , that seems to run, tried adding PHP 7.4 to the test but it didn't like that ¯\\_(ツ)_/¯

**Comment by chrs-myrs** [comment]
@xyNNN are you able to merge this so that I can stop using my forked version?

**Comment by xyNNN** [comment]
Release 3.1.0 is published, see https://packagist.org/packages/xynnn/google-tag-manager-bundle#3.1.0
Thanks @chrs-myrs

File: Resources/views/tagmanager_body.html.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <!-- Google Tag Manager (noscript) -->
-<noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ id }}"
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ id }}{{ url_suffix }}"
```
Comment (jkabat) [inline Resources/views/tagmanager_body.html.twig:2]: @chrs-myrs @xyNNN I wonder if this is going to work without |raw filter (& is escaped and rendered as &+amp;) Can you please investigate? I think it is not usable right now.

File: Resources/views/tagmanager_head.html.twig [hunk 2]
```diff
@@ -4,6 +4,6 @@
 <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
         new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
         j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl+'{{ url_suffix }}';f.parentNode.insertBefore(j,f);
```
Comment (jkabat) [inline Resources/views/tagmanager_head.html.twig:7]: @chrs-myrs @xyNNN I wonder if this is going to work without |raw filter (& is escaped and rendered as &+amp;) Can you please investigate? I think it is not usable right now.

**Pull request merged**